README.md

```diff
@@ -564,6 +564,69 @@ Expand them to see examples of:
   - [Query a list of all documents in a database](https://cloud.ibm.com/apidocs/cloudant?code=python#postalldocs)
   - [Query the database document changes feed](https://cloud.ibm.com/apidocs/cloudant?code=python#postchanges)
 
+### Model classes vs dictionaries
+
+This SDK supports 2 possible format to define an HTTP request. One approach uses only model classes, while the other only dictionaries.
+
+Here is an exmaple for the only model classes usage:
+
+```python
+price_index = SearchIndexDefinition(
+    index='function (doc) {  index(\"price\", doc.price);}',
```

Comment:
```suggestion
    index='function (doc){index(\"price\",doc.price);}',
```
or
```suggestion
    index='function (doc) { index(\"price\", doc.price); }',
```

**Other question:** Do we need `\"`s? Is `index='function (doc){index("price", doc.price);}',` invalid here?

Reply:
Changed in 921cccc